metricbeat/magefile.go

```diff
@@ -96,6 +98,27 @@ func Fields() error {
 	return mage.GenerateFieldsYAML("module")
 }
 
+func ExportDashboard() error {
+	module := os.Getenv("MODULE")
+	if module == "" {
+		return fmt.Errorf("MODULE must be specified")
+	}
+
+	id := os.Getenv("ID")
+	if module == "" {
+		return fmt.Errorf("Dashboad ID must be specified")
+	}
+
+	beatsDir, err := mage.ElasticBeatsDir()
+	if err != nil {
+		return err
+	}
+
+	file := path.Join(beatsDir, "metricbeat", "module", module, "_meta", "kibana", "6", "dashboard", id+".json")
```

Comment:
Use `filepath.Join`. You can write `filepath.Join(beatsDir, "metricbeat/module", module, "_meta/kibana/6/dashboard", id+".json")` and it will work fine because it always does a `filepath.Clean`.